Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+
+## Courses
+### Calculus
+
+The first macro-topic you need to be confident with is [Calculus](https://en.wikipedia.org/wiki/Calculus).
+
+It would be impossible to list how many Machine Learning system properties are based on its rules, like many of the scientific fields!
+
+The process of training an ML model it's nothing more than Calculus rules applied to statistical models! 
+
+**Lucky you!** 
+You can find great free courses offered by the professors of the [MIT](http://www.mit.edu/)(Massachusetts Institute of Technology), through the index page [MIT OpenCourseWare](https://ocw.mit.edu/index.htm).
+
+The course we suggest in particular is the following:
+
+[**Single Variable Calculus**](https://ocw.mit.edu/courses/mathematics/18-01sc-single-variable-calculus-fall-2010/)
+
+There are both the video lessons and the notes of the course.
+
+Once you've taken the full course, watch this awesome playlist from [3Blue1Brown](https://www.youtube.com/channel/UCYO_jab_esuFRV4b17AJtAw):
+
+[**The Essence of Calculus**](https://www.youtube.com/watch?v=WUvTyaaNkzM&list=PLZHQObOWTQDMsr9K-rj53DwVRMYO3t5Yr)
+
+Thanks to these two resources, you should be able to understand most of the mathematical reasons that underpin today's Machine Learning systems and build them to develop powerful Data Science projects
+
+If you want to dive deeper, here you can find a [Multivariable Calculus]() course with notes and videos, again from MIT:
+
+[**Multivariable Calculus**](https://ocw.mit.edu/courses/mathematics/18-02sc-multivariable-calculus-fall-2010/)
+
+Once you will feel ready to go deeper inside the mathematics for Machine Learning (definitely not take this at the first pass), here you find your Bible:
+
+[**Mathematics for Machine Learning**](https://mml-book.github.io/)
+
+#### Check your knowledge
+
+Test your knowledge with [**these**](http://webmath2.unito.it/paginepersonali/cordero/English/derivatives.pdf) exercises.
+
+Anyway you can find a ton of them online, and remember, that _the more you make exercise, the better!_
+
+### Linear Algebra and Matrix Algebra
+
+The second macro-topic on which Machine Learning systems are based is [linear algebra](https://en.wikipedia.org/wiki/Linear_algebra).
+
+You probably already tasted these topics in high school, but certainly not with the right degree of depth.
+
+So, how do you excellently learn linear algebra?
+
+Virgilio was born to spread knowledge and content, and there are many ways to do it: through text, audio, video... 
+
+But the absolute best way (in terms of effectiveness and speed of understanding) is the visual way!
+
+Our brain uses most of its computing power to process visual data, because when we evolved in the African Savannah the most important thing to do was to be able to see a predator and escape!
+
+It is therefore not surprising that the visual interaction of concepts installs them deep into our minds, almost experientially, rather than conceptually!
```

Comment:
> installs

I think the word you may be looking for is

> instill

Reply:
Nah I wanted to use "install" like "install a habit", like a Python package :D

Reply:
Fine it's your choice, but it might not be a way of saying it in English.